### ISSUE TITLE
Page size select element fails axe tests

### ISSUE THREAD

**Issue by bemailloux** [BrightspaceUILabs/pagination#7]
```
Error: Accessibility Violations
	---
	Rule: label-title-only
	Impact: serious
	Form elements should have a visible label (https://dequeuniversity.com/rules/axe/3.5/label-title-only?application=axeAPI)
	
	Issue target: d2l-insights-users-table,d2l-labs-pagination,select
	Context: <select class="d2l-input-select" title="Results per page">
	Fix all of the following:
	  Only title used to generate label for form element
```
